lib/glific/contacts.ex

```diff
@@ -691,4 +706,36 @@ defmodule Glific.Contacts do
   """
   @spec is_simulator_contact?(String.t()) :: boolean
   def is_simulator_contact?(phone), do: String.starts_with?(phone, @simulator_phone_prefix)
+
+  @doc """
+  create new contact histroy record.
+  """
+  @spec capture_history(Contact.t() | non_neg_integer(), atom(), map()) :: :ok
+  def capture_history(contact_id, event_type, attrs) when is_integer(contact_id) == true,
+    do:
+      get_contact!(contact_id)
+      |> capture_history(event_type, attrs)
+
+  def capture_history(%Contact{} = contact, event_type, attrs) do
+    ## I will add the telemetery evenets here.
+    attrs =
+      Map.merge(
+        %{
+          event_type: event_type |> Atom.to_string(),
+          contact_id: contact.id,
+          event_datetime: DateTime.utc_now(),
+          organization_id: contact.organization_id,
+          event_meta: %{}
+        },
+        attrs
+      )
+
+    %ContactHistory{}
+    |> ContactHistory.changeset(attrs)
+    |> Repo.insert()
+
+    :ok
```

Comment:
why not return the value from Repo.insert. we should stop returning :ok and ignore insert results